From 930e26a3ea9d223e04bac4cdde13697cec770031 Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Fri, 2 Nov 2012 18:25:08 +0100
Subject: [PATCH 175/278] x86: h264qpel: Only define mmxext QPEL functions if
 H264QPEL is enabled

This fixes compilation with --disable-everything and components enabled.
---
 libavcodec/x86/h264_qpel.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavcodec/x86/h264_qpel.c b/libavcodec/x86/h264_qpel.c
index f978520..cf944c1 100644
--- a/libavcodec/x86/h264_qpel.c
+++ b/libavcodec/x86/h264_qpel.c
@@ -1286,6 +1286,6 @@ QPEL16_OP(mc31, MMX)\
 QPEL16_OP(mc32, MMX)\
 QPEL16_OP(mc33, MMX)
 
-#if ARCH_X86_32 && HAVE_YASM // ARCH_X86_64 implies sse2+
+#if ARCH_X86_32 && HAVE_YASM && CONFIG_H264QPEL // ARCH_X86_64 implies SSE2+
 QPEL16(mmxext)
 #endif
-- 
1.7.9.5

